install-panel.sh

```diff
@@ -589,34 +638,18 @@ function centos8_dep {
 ## OTHER OS SPECIFIC FUNCTIONS ##
 #################################
 
-function ubuntu_universedep {
-  # Probably should change this, this is more of a bandaid fix for this
-  # This function is ran before software-properties-common is installed
-  apt update -y
-  apt install software-properties-common -y
-
-  if grep -q universe "$SOURCES_PATH"; then
-    # even if it detects it as already existent, we'll still run the apt command to make sure
-    add-apt-repository universe
-    echo "* Ubuntu universe repo already exists."
-  else
-    add-apt-repository universe
-  fi
-}
-
-function centos_php {
+centos_php() {
   curl -o /etc/php-fpm.d/www-pterodactyl.conf $GITHUB_BASE_URL/configs/www-pterodactyl.conf
 
   systemctl enable php-fpm
   systemctl start php-fpm
 }
 
-function firewall_ufw {
-  apt update
-  apt install ufw -y
+firewall_ufw() {
+  apt install -y ufw
 
   echo -e "\n* Enabling Uncomplicated Firewall (UFW)"
-  echo "* Opening port 22 (SSH), 80 (HTTP) and 443 (HTTPS)"
+  echo "* Opening port 22 (SSH), 80 (HTTP) and (optionaly) 443 (HTTPS)"
```

Comment:
```suggestion
  echo "* Opening port 22 (SSH), 80 (HTTP) and 443 (HTTPS)"
```

The word `optionaly` should be spelled `optionally` but I think we can remove it all together :slightly_smiling_face: